starknet_py/tests/e2e/client/full_node_test.py

```diff
@@ -104,3 +105,143 @@ async def test_get_storage_at_incorrect_address_full_node_client(full_node_clien
             key=get_storage_var_address("balance"),
             block_hash="latest",
         )
+
+
+# TODO FIX TESTS BELOW AFTER DEVNET FIX REGARDING CONTINUATION_TOKEN
+# more about in /starknet_py/net/full_node_client.py:162
+
+FUNCTION_ONE_NAME = "put"
+EVENT_ONE_NAME = "put_called"
+FUNCTION_TWO_NAME = "another_put"
+EVENT_TWO_NAME = "another_put_called"
+
+
+@pytest.mark.run_on_devnet
+@pytest.mark.asyncio
+async def test_get_events_with_single_event_no_continuation_token(
+    full_node_client,
+    simple_storage_with_event_contract: Contract,
+):
+    await simple_storage_with_event_contract.functions[FUNCTION_ONE_NAME].invoke(
+        1, 1, auto_estimate=True
+    )
+    await simple_storage_with_event_contract.functions[FUNCTION_ONE_NAME].invoke(
+        2, 1, auto_estimate=True
+    )
+    await simple_storage_with_event_contract.functions[FUNCTION_ONE_NAME].invoke(
+        3, 1, auto_estimate=True
+    )
+    await simple_storage_with_event_contract.functions[FUNCTION_ONE_NAME].invoke(
+        4, 1, auto_estimate=True
+    )
+    events_response = await full_node_client.get_events(
+        from_block_number=0,
+        to_block_hash="latest",
+        address=simple_storage_with_event_contract.address,
+        keys=[EVENT_ONE_NAME],
```

Comment:
Instead of doing keys parsing in the client, you should be doing it when calling this method. Perhaps `get_selector_from_name` should be used here (or similiar method should be introduced? But I don't like this idea).

cc @war-in @drknzz

Reply:
```suggestion
        keys=[get_selector_from_name(EVENT_ONE_NAME)],
```

Reply:
`get_selector_from_name` would be enough 👌